challenges/panda-api.md

```diff
@@ -0,0 +1,37 @@
+# PandaScore API
```

Comment:
This name is no good. The objective of this challenge is to create a web app to consume data from eSports sites. Panda Score is just the suggested API, there are others. The name should be something like *Challenge: eSport Analysis & Statistics App*.